solidity/truffle-examples/random-datasource/README.md

```diff
@@ -0,0 +1,71 @@
+# :wrench: :construction: Testing Oraclize's Random Example.
+
+&nbsp;
+
+This repo is to demonstrate how you would set up an Oraclize smart-contract development environment using Truffle to do most of the heavy lifting for you. Head on over to the `./test` folder to examine the javascript files that thoroughly test the smart-contract, which latter you will find in `./contracts`.
+
+## :page_with_curl:  _Instructions_
+
+Since the random datasource is currently available on the Ethereum mainnet and on some Ethereum public testnets only (Rinkeby, Kovan, and Ropsten Revival); and it is *not integrated yet with private blockchains/testrpc/browser-solidity-vmmode*, you will need to **get a mnemonic passphrase and an infura key** to make the example work.
+
+**A)** Go on the infura website to get an infura key: [infura.io](https://infura.io).
+The key is a 32 HEX characters string
+
+**B)** Get a mnemonic passphrase, an easy way is to get the Metamask one:
+[metamask.io](https://metamask.io/)
+
+**C)** In `ethereum-examples/solidity/truffle-examples/random-datasource`, create a new file `apikeys.js` or `.env` and *add the mnemonic passphrase and the infura key to it*, such as:
```

Comment:
These instructions come _before_ the main instructions, which latter's first command involves cloning this repo. And so it's confusing, since this earlier line assumes the user has _already_ cloned the repo. 

It's probably best to merge to two instruction sets into one.